packages/react-contextual-pane/stories/Header/index.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import type { Meta } from '@storybook/react';
+import { Header } from '@fluentui-contrib/react-contextual-pane';
+import { getNovaDecorator } from '@nova/react-test-utils';
+import { buildSchema } from 'graphql';
+export { Default } from './Default.stories';
+
+const meta = {
+  title: 'Packages/react-contextual-pane/Header',
+  component: Header,
+  decorators: [getNovaDecorator(buildSchema(``))],
```

Comment:
With this change I'm able to run this storybook

```suggestion
  decorators: [getNovaDecorator(buildSchema(`type Query { noop: Boolean }`))],
```